### PR TITLE
Environment variables [8581]

### PR DESCRIPTION
**Pull request by IkerLuengo**
Moved from overview to its own section.

### CODE REVIEW

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/482/)

File: docs/fastdds/env_vars/env_vars.rst [hunk 1]
```diff
@@ -2,3 +2,12 @@
 
 Environment variables
 =====================
+
+This is the list of all environment variables that affect the behavior of *Fast DDS*:
```
Comment (rsanchez15) [inline docs/fastdds/env_vars/env_vars.rst:6]: It doesn't seem right to say it's "the list of all the..." and then only have one.

File: docs/fastdds/env_vars/env_vars.rst [hunk 2]
```diff
@@ -2,3 +2,26 @@
...
+    When this variable is set, it automatically activates the :ref:`Server-Client Discovery<discovery_server>`.
+    The value of the variable must represent the locator of the server,
+    in the form of the IP address (e.g., '192.168.2.23') or address-port pair (e.g., '192.168.2.23:24353').
+    If no port is specified, the default port 11311 is used.
+
+    If the specified address is a local address of the current machine, it will start the
+    :ref:`dds_layer_domainParticipant` as a server.
+    If the address is a remote one, or if there is already a server at the given port,
+    it will start the :ref:`dds_layer_domainParticipant` as a client,
+    connected to the server at the specified locator.
```
Comment (EduPonz) [inline docs/fastdds/env_vars/env_vars.rst:23]:
```suggestion
    connecting to the server at the specified locator.
```

File: docs/fastdds/env_vars/env_vars.rst [hunk 3]
```diff
@@ -2,3 +2,26 @@
 
 Environment variables
 =====================
+
```
Comment (EduPonz) [inline docs/fastdds/env_vars/env_vars.rst:5]: May I be so bold as to ask for some snippets (windows and linux) to see then in action?

Comment (IkerLuengo) [reply docs/fastdds/env_vars/env_vars.rst:5]: Are we assuming a bash console on windows? If we are, both examples would be the same, (except the file name conventions). I gave it a try, you tel me if you like it.

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/490/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/495/)

File: docs/fastdds/env_vars/env_vars.rst [hunk 4]
```diff
@@ -10,6 +10,21 @@ This is the list of environment variables that affect the behavior of *Fast DDS*
...
+| .. code-block::                                              |
+|                                                              |
+|    FASTRTPS_DEFAULT_PROFILES_FILE=/home/user/profiles.xml    |
++--------------------------------------------------------------+
+| **windows**                                                  |
++--------------------------------------------------------------+
+| .. code-block::                                              |
+|                                                              |
+|    set FASTRTPS_DEFAULT_PROFILES_FILE=C:\profiles.xml        |
++--------------------------------------------------------------+
```
Comment (EduPonz) [inline docs/fastdds/env_vars/env_vars.rst:25]: I'd indent these tables so they are aligned with the text.

File: docs/fastdds/env_vars/env_vars.rst [hunk 5]
```diff
@@ -10,6 +10,21 @@ This is the list of environment variables that affect the behavior of *Fast DDS*
     If this variable is set and its value corresponds with an existing file, *Fast DDS* will load its profiles.
     For more information about XML profiles, please refer to :ref:`xml_profiles`.
 
++--------------------------------------------------------------+
+| **linux**                                                    |
++--------------------------------------------------------------+
+| .. code-block::                                              |
+|                                                              |
+|    FASTRTPS_DEFAULT_PROFILES_FILE=/home/user/profiles.xml    |
```
Comment (EduPonz) [inline docs/fastdds/env_vars/env_vars.rst:18]: Just for consistency with `set`
```suggestion
|    export FASTRTPS_DEFAULT_PROFILES_FILE=/home/user/profiles.xml    |
```

File: docs/fastdds/env_vars/env_vars.rst [hunk 6]
```diff
@@ -20,7 +35,22 @@ This is the list of environment variables that affect the behavior of *Fast DDS*
...
     it will start the :ref:`dds_layer_domainParticipant` as a client,
-    connected to the server at the specified locator.
+    connecting to the server at the specified locator.
+
++--------------------------------------------------------------+
+| **linux**                                                    |
++--------------------------------------------------------------+
+| .. code-block::                                              |
+|                                                              |
+|    ROS2_AUTO_CLIENT_SERVER=192.168.2.23:24353                |
```
Comment (EduPonz) [inline docs/fastdds/env_vars/env_vars.rst:45]:
```suggestion
|    export ROS2_AUTO_CLIENT_SERVER=192.168.2.23:24353                |
```

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/496/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/497/)

**Pull request merged**